azkaban-web-server/src/main/java/azkaban/webapp/AzkabanWebServer.java

```diff
@@ -681,18 +671,28 @@ public static void main(String[] args) throws Exception {
     StdOutErrRedirect.redirectOutAndErrToLog();
 
     logger.info("Starting Jetty Azkaban Web Server...");
-    Props azkabanSettings = AzkabanServer.loadProps(args);
+    Props props = AzkabanServer.loadProps(args);
 
-    if (azkabanSettings == null) {
+    if (props == null) {
       logger.error("Azkaban Properties not loaded. Exiting..");
       System.exit(1);
     }
 
-    final Server server = createServer(azkabanSettings);
+    /* Initialize Guice Injector */
+    SERVICE_PROVIDER.setInjector(Guice.createInjector(
+        new AzkabanCommonModule(props),
+        new AzkabanWebServerModule()
+    ));
 
-    app = new AzkabanWebServer(server, azkabanSettings);
+    launch(props);
+  }
+
+  public static void launch(Props azkabanSettings) throws Exception {
```

Comment:
This is being called by the solo server main. The solo server used the call the `main` methods for both web and exec servers.